templates/enmasse-operator/050-Deployment-enmasse-operator.yaml

```diff
@@ -73,6 +73,8 @@ spec:
           value: ${KEYCLOAK_IMAGE}
         - name: KEYCLOAK_PLUGIN_IMAGE
           value: ${KEYCLOAK_PLUGIN_IMAGE}
```

Comment:
You need to add all the console images as envs to the operator so that it can be overridden from env